app/graphql/types/query_type.rb

```diff
@@ -224,5 +224,13 @@ def method_missing(name, *args)
     resolved_field :applicant, Types::ApplicantType, null: false do
       argument :applicant_id, ID, required: true
     end
+
+    resolved_field :user_standings, [Types::UserStandingType], null: false do
+      argument :student_id, ID, required: true
+    end
+
+    resolved_field :standings, [Types::StandingType], null: false
```

Comment:
Shouldn't we paginate the standings? 🤔 
Else we should had a safe limit of total number of standings that can be created for a user;

Reply:
Added the rate limit for the number of user_standings or Standings logs that can be created for a user - 100

https://github.com/pupilfirst/pupilfirst/pull/1468/commits/faf838e0b8d2ca5fe23d765b5ccccdb949da0d64